### ISSUE TITLE
aria/menuitem_role `menuitem` is not announced by NVDA and FF

### ISSUE THREAD

**Issue by anevins12** [accessibilitysupported/a11ysupport.io#286]
I noticed that NVDA and FF doesn't announce the `menuitem` role when navigating to these controls in the APG "actions" `menu` button example: https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/examples/menu-button-actions-active-descendant/

I thought the test should be updated because it says NVDA and FF has full support for the `menuitem` role: https://a11ysupport.io/tech/aria/menuitem_role

**Comment by scottaohara** [comment]
having full support for the role doesn't necessarily mean the role _has_ to be announced by screen readers.   This isn't even specific to firefox, nvda doesn't announce the menuitem role when paired with chrome either.  and this is not unique to menuitems, similar occurs with options within a listbox with some screen readers.   rational there being that a user is within a composite widget that specifically contains menuitems or options... so the screen reader makes the choice to reduce verbosity

**Comment by anevins12** [comment]
I appreciate the explanation
